portfolio/src/main/webapp/index.html

```diff
@@ -40,23 +40,27 @@
 <body>
 
 <ul>
-  <li><a class="active" href="#home">Home</a></li>
-  <li><a href="#about">About</a></li>
-  <li><a href="#current">Work History</a></li>
-  <li><a href="#connect">Let's Connect!</a></li>
+  <li><a class="active" href="#home"; href="./index.html">Home</a></li>
+  <li><a href="#about"; href="./about.html">About</a></li>
+  <li><a href="#experience"; href="./experience.html">Experience</a></li>
```

Comment:
What is the purpose of href='#Home' etc? Is that an Id?

Reply:
I was trying to make it an id but realized I did it wrong. I fixed it in the newest commit!

Reply:
Done